test-dependencies.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-PIP3='sudo pip3 install -q -U --no-cache'
+PIP3='sudo pip3 install -q -U'
```

Comment:
Hmm, I have to check this, but iirc `sudo pip` causes trouble if you don't use `--no-cache`; I think it was something about it potentially creating a cache in your home dir owned by root.